clients/unshaded/src/main/java/tachyon/client/file/options/UnmountOptions.java

```diff
@@ -16,47 +16,40 @@
 package tachyon.client.file.options;
 
 import tachyon.annotation.PublicApi;
-import tachyon.conf.TachyonConf;
 
 /**
  * Method option for unmounting a path.
  */
 @PublicApi
 public final class UnmountOptions {
+  /** Whether the data under the mount should be synchronously freed from Tachyon, currently
```

Comment:
I originally wanted to have some field in all the options, but looking at it now it seems like a bad idea.